### ISSUE TITLE
[Bug] Album art in folder not imported

### ISSUE THREAD

**Issue by werdahias** [nate-xyz/resonance#36]
In lollypop the album art can either come from

1) the metadata itself or
2) an image in the folder

All my music is organized with 2) because that way the files are separated from the artwork and you can easily fix a wrong one.
Right now the image in the folder doesn't get recognized by Resonance. It'd be nice it this were supported. lollypop also supports crawling the web for artwork and downloads the selected one in the album tracks' folder.

**Comment by nate-xyz** [comment]
Do you know if there's some kind of de facto standard for what album art is named in the folder? Typically I see cover.whatever but sometimes I see folder.whatever . Parsing the image file names becomes an issue when there's more than one cover, additional art or artist images in the folder as well.

**Comment by nate-xyz** [comment]
I could definitely also implement crawling for art, I already have this functionality for MPRIS and discord art by looking up the MusicBrainz ID for a release and then using the cover art archive: https://coverartarchive.org/, but maybe I could scrape discogs for higher quality art?

**Comment by werdahias** [comment]
I think lollypop uses cover.(jpg/png). 
The exact implementation is here: 

https://gitlab.gnome.org/World/lollypop/-/blob/master/lollypop/artwork_album.py

Maybe this helps

**Comment by nate-xyz** [comment]
Yeah I think I'll probably just only look for cover.(jpg|png) in order to mitigate the complexity of trying to determine the right file

**Comment by DWW256** [comment]
Also include cover.jpg|png and Folder.jpg|png.  And you should also import gif, webp, and maybe even avif if the libraries you're using support them.
